### PR TITLE
fix(workflow): Shortening lock key

### PR DESCRIPTION
**Pull request by iProgramStuff**
refer to branch name

### CODE REVIEW

File: src/sentry/tasks/post_process.py [hunk 1]
```diff
@@ -277,10 +277,7 @@ def post_process_group(
...
 
             try:
-                lock = locks.get(
-                    "workflow-owners-ingestion:org-{}-has-group-lock".format(event.group_id),
-                    duration=10,
-                )
+                lock = locks.get("w-o:{}-d-l".format(event.group_id), duration=10,)
                 try:
                     with lock.acquire():
                         has_commit_key = "workflow-owners-ingestion:org-{}-has-commits".format(
```
Comment (wedamija) [inline src/sentry/tasks/post_process.py:283]: While we're here, should we shorten this too?

**Review by wedamija — APPROVED** [review]

File: src/sentry/tasks/post_process.py (hunk 1 above)
Comment (iProgramStuff) [reply src/sentry/tasks/post_process.py]: Yes
